concepts/src/controllers/concept.ts

```diff
@@ -0,0 +1,53 @@
+import { Client as ElasticClient } from "@elastic/elasticsearch";
+import { RequestHandler } from "express";
+import asyncHandler from "express-async-handler";
+import { Concept } from "../types";
+import { HttpError } from "./error";
+
+type PathParams = { id: string };
+
+type ConceptHandler = RequestHandler<PathParams, Concept>;
+
+type Dependencies = { elasticClient: ElasticClient; index: string };
+
+const conceptController = ({
+  elasticClient,
+  index,
+}: Dependencies): ConceptHandler =>
+  asyncHandler(async (req, res) => {
+    const id = req.params.id;
+    const worksIndexResponse = await elasticClient.search({
+      index,
+      body: {
+        size: 1,
+        _source: ["display.subjects"],
+        query: {
+          term: {
+            "query.subjects.id": id,
+          },
+        },
+      },
+    });
```

Comment:
Doing this with Elasticsearch is fine now you've done it, but FWIW, my initial plan was to proxy responses from the public API (hence why I added the subject ID filtering).

Reply:
Yeah, I realised that after I'd done it and am not sure why I thought ES was necessary - on the other hand, this does make iteration a bit easier.